mf4-csv.py

```diff
@@ -75,58 +67,33 @@ def file_convert():
         #     with open(file+'.ChannelGroup_' + str(x) + '.csv') as f:
         #         first_line = f.readline()
     else:
-        print("You fool; thats not a valid file! The file must be an .mf4")
+        print("You fool; thats not a valid file! The file must be a .mf4")
     handle_data(files)
 
+
+#Streams data into the DB 
 def handle_data(files):
     for file in files:
         with open(file) as open_file:
-            first_line = open_file.readline()
-            first_line_listed = first_line.split(",")
-            status = "CAN_DataFrame.CAN_DataFrame.ID" in first_line_listed
-            dataLengthIndex = first_line_listed.index("CAN_DataFrame.CAN_DataFrame.DataLength")
-            dataBytesIndex = first_line_listed.index("CAN_DataFrame.CAN_DataFrame.DataBytes")
-            idIndex = first_line_listed.index("CAN_DataFrame.CAN_DataFrame.ID")
-            counter = 0
             for line in open_file:
-                counter += 1
-                if(status):
-                    outboard, cob_id = determine_board_owner(line, idIndex)
-                    extract_data_bytes(outboard, cob_id, dataLengthIndex, dataBytesIndex)
-
-
-
-
-
-def determine_board_owner(line, index):
-    listed_line = line.split(",")
-    id = int(listed_line[index])
-    """Values based on the CAN Open standard. Each hex value relates to a TPDO message"""
-    if(id < tpdoDictionary["TPDO0"]):
-        return("Unknown COBID", 0)
-    if(id <= tpdoDictionary["DEADZONE_1"]):
-        default_COB_ID = tpdoDictionary["TPDO0"]
-    elif(id <= tpdoDictionary["TPDO1"]):
-        return("Unknown COBID", tpdoDictionary["DEADZONE_1"])
-    elif(id <= tpdoDictionary["DEADZONE_2"]):
-        default_COB_ID = tpdoDictionary["TPDO1"]
-    elif(id <= tpdoDictionary["TPDO1"]):
-        return("Unknown COBID", tpdoDictionary["DEADZONE_2"])
-    elif(id <= tpdoDictionary["DEADZONE_3"]):
-        default_COB_ID = tpdoDictionary["TPDO2"]
-    elif(id <= tpdoDictionary["TPDO3"]):
-        return("Unknown COBID", tpdoDictionary["DEADZONE_3"])
-    elif(id <= tpdoDictionary["DEADZONE_4"]):
-        default_COB_ID = tpdoDictionary["TPDO3"]
-    else:
-        return("Unknown COBID", tpdoDictionary["DEADZONE_4"])
-    return(nodeIDDictionary[str(id-default_COB_ID)], default_COB_ID)
+                canMessageArr = line.split(',')
+                formattedArray = format_data_bytes(canMessageArr[6])
+                #Currently hard coding the contextId input because there is only one (dummy) contextId
```

Comment:
```suggestion
                #TODO Currently hard coding the contextId input because there is only one (dummy) contextId
```
So it's easy to find later